src/AppInstallerCLICore/ExecutionContext.cpp

```diff
@@ -116,14 +116,12 @@ namespace AppInstaller::CLI::Execution
             // Unless we want to spin a separate thread for all work, we have to just exit here.
             if (m_CtrlSignalCount >= 2)
             {
+                Logging::Telemetry().LogCommandTermination(hr, file, line);
                 std::exit(hr);
             }
         }
 
-        if (!file.empty() && line)
-        {

```

Comment:
Why are these removed?  I can't remember why I added them, but I don't think it was for no reason (it actually might have been to prevent the first CTRL-C from being logged).

Reply:
yes. The initial intention is to not log ctrl_c. But we want ctrl_c to be logged explicitly to reduce noise

---
In reply to: [518297562](https://github.com/microsoft/winget-cli/pull/634#discussion_r518297562) [](ancestors = 518297562)